v3/src/components/data-display/components/legend/categorical-legend-model.ts

```diff
@@ -0,0 +1,254 @@
+import { makeAutoObservable } from "mobx"
+import { measureText } from "../../../../hooks/use-measure-text"
+import { logMessageWithReplacement } from "../../../../lib/log-message"
+import { missingColor } from "../../../../utilities/color-utils"
+import { axisGap } from "../../../axis/axis-types"
+import { getStringBounds } from "../../../axis/axis-utils"
+import vars from "../../../vars.scss"
```

Comment:
This is an interesting case. I would normally put this at the bottom given that it's a CSS import, but since it's being used as a JavaScript import in this case, it could arguably be placed here as well. 🤷

Reply:
I'll move it down because it does look strange and I wasn't paying attention to it being a css file.